.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Thanks for prettier config. :+1:

Reply:
No prob!  Meant to ask you if you wanted to use Prettier....

Reply:
If you want I can put that in a separate PR so we don't have a bunch of formatting changes mixed in here.

Reply:
No, there are small number of changed line.